src/puppetlabs/trapperkeeper/services/authorization/authorization_core.clj

```diff
@@ -56,6 +56,13 @@
   (->> (select-keys config-map #{:allow :allow-ip :deny :deny-ip})
        (reduce #(add-individual-acl (first %2) (second %2) %1) rule)))
 
+(defn add-query-params
+  "Add any query parameters specified in configuration to the rule."
+  [rule {:keys [query-params]}]
+  (if query-params
```

Comment:
Do we need this if statement?  Can we just do the `reduce-kv` unconditionally and have it effectively return back the original rule untouched in the case that `query-params` was not in the input map?